manifests/package.pp

```diff
@@ -106,4 +112,16 @@
     require => Package[$container_package],
   }
 
+  if $experimental_filters == true {
+    package { $repose::params::experimental_filters_packages:
+      ensure => $package_ensure, 
+      require => Package[$container_package],
+    }
+  } else {
+    package { $repose::params::experimental_filters_packages:
+      ensure => absent, 
```

Comment:
My only concern here is if someone had the experimental filters package installed manually, this would now remove that package.